code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1152,13 +1153,13 @@
 	if((temp_turf.z != our_turf.z) || M.stat!=CONSCIOUS) //Not on the same zlevel as us or they're dead.
 //		to_chat(world, "[(temp_turf.z != our_turf.z) ? "not on the same zlevel as [M]" : "[M] is not concious"]")
 		if(temp_turf.z != 2)
-			to_chat(src, "The mind of [M] is too faint...")//Prevent "The mind of Admin is too faint..."
-
-
+			if(prob(5))//Can't tell who is missing as easily as before. I feel like this is a fun alternative to removing it altogether
+				to_chat(src, "The mind of [M] is too faint...")//Prevent "The mind of Admin is too faint..."
 		return 0
 	if(M_PSY_RESIST in M.mutations)
 //		to_chat(world, "[M] has psy resist")
-		to_chat(src, "The mind of [M] is resisting!")
+		if(prob(2))
```

Comment:
This isn't a good idea, because you're going to get an issue a la "why does psy do nothing"
Keep some kind of failure message.